src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultArr = [];
+  const clone = Object.assign({}, state);
+
+  for (const action in actions) {
+    const actionType = actions[action].type;
+    const removeArr = actions[action].keysToRemove;
+    const extraData = actions[action].extraData;
```

Comment:
try to use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here

Reply:
@Bohdan-Maksymiuk done